.tekton/cli-build.yaml

```diff
@@ -461,11 +469,11 @@ spec:
           - name: kind
             value: task
         resolver: bundles
-      params:
-      - name: image-url
-        value: $(tasks.build-image-index.results.IMAGE_URL)
-      - name: image-digest
-        value: $(tasks.build-image-index.results.IMAGE_DIGEST)
+      when:
+        - input: $(params.skip-checks)
+          operator: in
+          values:
+            - "false"
```

Comment:
Not sure why the `when` and `params` swap locations, but this matches the current default pipeline.